src/layers/src/cluster-layer/cluster-layer-icon.tsx

```diff
@@ -20,7 +20,7 @@
 
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import Base from 'components/common/icons/base';
+import {Base} from '@kepler.gl/constants';
```

Comment:
can you make a copy of `Base` and put it in`layers/src/react` and keep the `Base` in `components/icons`? so that we don't move it to constant?

Reply:
`Base` is imported by every other icon. So if we want at less one icon to be in `@kepler.gl/constants` then we need `Base` to be there as well